pkg/virt-controller/services/template.go

```diff
@@ -499,6 +499,29 @@ func (t *templateService) renderLaunchManifest(vmi *v1.VirtualMachineInstance, i
 			}
 			sidecarVolumes = append(sidecarVolumes, vol)
 		}
+		if requestedHookSidecar.PVC != nil {
+			pvc, err := t.virtClient.CoreV1().PersistentVolumeClaims(vmi.Namespace).Get(context.TODO(), requestedHookSidecar.PVC.Name, metav1.GetOptions{})
```

Comment:
@vasiliy-ul no it isn't mandatory, but what if the PVC doesn't exist?

Reply:
If it does not exist, then the initial creation of the VMI will fail (I assume, virt-launcher pod will not start). But then it will succeed later after reconciliation when the corresponding PVC appears, no?